lib/levels.js

```diff
@@ -180,6 +203,20 @@ function assertNoLevelCollisions (levels, customLevels) {
   }
 }
 
+/**
+ * @description Validates whether levelComparison is correct
+ * @throws Error
+ * @param {SORTING_ORDER | Function} levelComparison
+ * @returns
+ */
+function assertLevelComparison (levelComparison) {
+  if (Object.values(SORTING_ORDER).includes(levelComparison) || typeof levelComparison === 'function') {
+    return
+  }
```

Comment:
I think this might be more efficient:
```suggestion
  if (typeof levelComparison === 'function') {
  		return
  }
  if (typeof levelComparison === 'string' && Object.prototype.hasOwnProperty.call(SORTING_ORDER, levelComparison)) {
    return
  }
```

Reply:
Added suggested changes